File: CodeForces/101-200/118/A/118A.py
# Problem 118A: String Task
# https://codeforces.com/contest/118/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 26th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

string = str(input())

string = string.lower()

new_string = ""

for character in string:
    if character in ["a","e","i","o","u","y"]:
        continue
    new_string += f".{character}"

print(new_string)

File: CodeForces/101-200/118/A/118A.rb
# Problem 118A: String Task
# https://codeforces.com/contest/118/problem/A
# Executed with: ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [x86_64-linux-gnu]
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: October 22nd, 2025
# Codeforces language used: Ruby 3.2.2

w = gets
w = w.chomp.downcase.chars

ans = ""

w.each do |character|
  if !(character == "a" || character == "e" || character == "i" || character == "o" || character == "u" || character == "y")
    ans += "."
    ans += character
  end
end

puts ans
